api/source/basics.rst

```diff
@@ -99,232 +99,419 @@ Languages
+| Korean                                   | ko                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Malay                                    | ms                  | ❌                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Norwegian                                | no                  | ❌                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Polish                                   | pl                  | ❌                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Portuguese                               | pt                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Portuguese (Portugal)                    | pt-pt               | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Russian                                  | ru                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Spanish                                  | es                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Spanish (Spain)                          | es-es               | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Swedish                                  | se                  | ❌                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Turkish                                  | tr                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Ukrainian                                | uk                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Vietnamese                               | vn                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+
+
+
+Import
```

Comment:
This is not needed as we don't do any processing here, so we support all languages.